site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,32 @@
+---
+name: Box shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box shadow
+
+<LargeParagraph>Shadows can be used under certain elements to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or otherwise remain unnoticed.
```

Comment:
Grammarly thinks this sentence contains redundancy (otherwise):

<img width="1341" alt="image" src="https://user-images.githubusercontent.com/2777633/153359325-aad19a97-78f9-4594-ac35-c5f570277285.png">